catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardImpl.java

```diff
@@ -110,7 +113,7 @@ public MetacardImpl(MetacardType type) {
          * serialized object is maintained. For instance, if a null check is added in the
          * constructor, the same check should be added in the readObject() method.
          */
-        map = new HashMap<String, Attribute>();
+        map = new ConcurrentHashMap<>();
```

Comment:
Fundamental issues, yes. But I would think we'd need wrapping semantics to allow people to start using something like them instead of regular MetacardImpl... I guess we could leave it out and say its a tradeoff ¯\_(ツ)_/¯. sorry for turning your PR into a random idea ground @jlcsmith  XD